packages/dds/tree/src/test/simple-tree/unhydratedNode.spec.ts

```diff
@@ -330,7 +331,8 @@ describe("Unhydrated nodes", () => {
 		assert.throws(
 			() => new TestArray([leaf, leaf]),
 			(e: Error) =>
-				validateAssertionError(e, /A node may not be in more than one place in the tree/),
+				e instanceof UsageError &&
```

Comment:
We have a validateUsageError in our test/utils that is a more concise way to check this.